sys/ofed/drivers/infiniband/core/ib_umem.c

```diff
@@ -180,7 +180,7 @@ struct ib_umem *ib_umem_get(struct ib_ucontext *context, unsigned long addr,
 	sg_list_start = umem->sg_head.sgl;
 
 	while (npages) {
-		ret = get_user_pages(cur_base,
+            ret = get_user_pages((void *)cur_base,
```

Comment:
This line is now spaces?

Reply:
Other way around, but I'll fix it.

Reply:
Well, it did go from tabs to spaces oddly enough.  Not sure why, but fixed.